x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/attach_index_box.tsx

```diff
@@ -65,66 +65,109 @@ export const AttachIndexBox: React.FC<AttachIndexBoxProps> = ({ connector }) =>
     createApiError,
     attachApiError,
   } = useValues(AttachIndexLogic);
+
+  const { makeRequest } = useActions(FetchAvailableIndicesAPILogic);
+  const { data, status } = useValues(FetchAvailableIndicesAPILogic);
+  const isLoading = [Status.IDLE, Status.LOADING].includes(status);
+
+  // Helper function to remove the managed connector index prefix from the index name
+  const removePrefixConnectorIndex = (indexName: string) => {
```

Comment:
We need this to hack around the combo box input state, we don't have sufficient access to inject the `content-` so I had to wrap some logic with the `removePrefixConnectorIndex` and `prefixConnectorIndex`